frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.component.sass

```diff
@@ -47,4 +47,8 @@
     display: inline-block
     overflow: hidden
     white-space: nowrap
-    text-overflow: ellipsis
\ No newline at end of file
+    text-overflow: ellipsis
+
+  &--option-principal-wrapper
+    margin-right: 5px
+    float: left
```

Comment:
`float` is a bad practice for styling inline, since it is originally intended to separate independent parts of the application. Please use `grid` or `flex` for this use case. You can read more on `float` [here](https://stackoverflow.com/a/9777248/8900797) and [here](https://css-tricks.com/is-css-float-deprecated/).